From 08e37a624c7d938a0401b216c22542ee6b59dffb Mon Sep 17 00:00:00 2001
From: Ting-Wei Lan <lantw@src.gnome.org>
Date: Tue, 14 Jan 2020 21:05:08 +0800
Subject: [PATCH] Revert "Change default pkgconfig directory on FreeBSD"

This reverts commit aba8792b6657e3aa380586764a8b3ec567895a28.

The commit aba8792b6657e3aa380586764a8b3ec567895a28 changes the
installation path of pkg-config files from lib/pkgconfig to
libdata/pkgconfig on FreeBSD. Since JHBuild does not change the
directory layout depending on the platform, revert it for now to keep
the current PKG_CONFIG_PATH working.

Works around https://github.com/mesonbuild/meson/pull/4410.
---
 mesonbuild/modules/pkgconfig.py | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/mesonbuild/modules/pkgconfig.py b/mesonbuild/modules/pkgconfig.py
index a37dd4f6..cf86785a 100644
--- a/mesonbuild/modules/pkgconfig.py
+++ b/mesonbuild/modules/pkgconfig.py
@@ -437,10 +437,7 @@ class PkgConfigModule(ExtensionModule):
         pcfile = filebase + '.pc'
         pkgroot = kwargs.get('install_dir', default_install_dir)
         if pkgroot is None:
-            if mesonlib.is_freebsd():
-                pkgroot = os.path.join(state.environment.coredata.get_option(mesonlib.OptionKey('prefix')), 'libdata', 'pkgconfig')
-            else:
-                pkgroot = os.path.join(state.environment.coredata.get_option(mesonlib.OptionKey('libdir')), 'pkgconfig')
+            pkgroot = os.path.join(state.environment.coredata.get_option(mesonlib.OptionKey('libdir')), 'pkgconfig')
         if not isinstance(pkgroot, str):
             raise mesonlib.MesonException('Install_dir must be a string.')
         self.generate_pkgconfig_file(state, deps, subdirs, name, description, url,
-- 
2.24.1

